search/query.go

```diff
@@ -92,7 +89,6 @@ func doSearch(ctx context.Context, escli *es.Client, index string, query interfa
 		escli.Search.WithIndex(index),
 		escli.Search.WithBody(&buf),
 		escli.Search.WithTrackTotalHits(true),
-		escli.Search.WithSize(30),
```

Comment:
Testing on the ES I have (8.4.0), this parameter overrides anything in the query body, so I've pushed this into the posts or profile query generators.